packages/wrangler/src/publish.ts

```diff
@@ -502,7 +504,7 @@ See https://developers.cloudflare.com/workers/platform/compatibility-dates for m
 			compatibility_flags:
 				props.compatibilityFlags ?? config.compatibility_flags,
 			usage_model: config.usage_model,
-			keep_bindings: true,
+			keep_bindings: keepVars,
```

Comment:
Given that this is a boolean then I think we should actually rename this property to `keepVars` too.
We should only switch to `keep_bindings` at the point when we convert from boolean to the array of bindings to keep.

Reply:
Ok! I was thinking about that too, the `keep_bindings` seemed out of place with these changes but I wasn't sure of a approach, I agree with this.

Reply:
Yeah same lol